client/src/components/exercise/lab11/pages/LiteracyExerciseEnd.js

```diff
@@ -0,0 +1,81 @@
+import { navigate } from "@reach/router";
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import { EXERCISE_PLAYING, LAB_ID } from "../../../../constants/lab11";
+import UserLabService from "../../../../services/UserLabService";
+
+const LiteracyExerciseEnd = (props) => {
+  const { actions, user } = props;
+
+  const handleFinish = () => {
+    actions.updateState("EXERCISE_IDLE");
+    UserLabService.complete_exercise(LAB_ID);
+    if (user?.firstname !== null && user !== null) {
+      UserLabService.user_complete_exercise(user.userid, LAB_ID);
+    }
+    navigate("/Lab11/Reinforcement");
+  };
+
+  useEffect(() => {
+    actions.updateState(EXERCISE_PLAYING);
+  }, []);
+
+  const handleTryAgain = () => {
+    navigate("/Lab11/Exercise/InformationLetterFogIndexFormula");
+  };
+
+  return (
+    <div className="center-div">
+      <div className="playthrough__sentence">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi eros
+        tellus, hendrerit quis iaculis non, blandit eget turpis. Sed at
+        tristique ex. Nam feugiat sodales nulla, ac lobortis diam porttitor vel.
+        Orci varius natoque penatibus et magnis dis parturient montes, nascetur
+        ridiculus mus. Sed iaculis lectus orci, eget imperdiet lectus venenatis
+        non. Pellentesque tincidunt arcu a nisi luctus, non vulputate elit
+        placerat. Sed eu eleifend nunc, at porttitor massa. Vestibulum eleifend
+        turpis vitae pulvinar consequat. Nulla non dui magna. Curabitur vitae
+        lacinia ante, non convallis turpis. Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit. Cras ut dolor velit. Mauris posuere, enim
+        vitae feugiat fringilla, eros quam iaculis est, in facilisis massa odio
+        vel eros. Sed ut erat convallis, posuere lorem sit amet, auctor augue.
+        Sed in neque sed magna aliquam semper. Donec tincidunt risus at nibh
+        dictum consectetur. Morbi porta fringilla mauris a cursus. Sed
+        condimentum porttitor mauris, eget egestas libero eleifend et. Donec
+        laoreet congue tellus id condimentum. Sed accumsan non nisl non
+        tincidunt. Suspendisse potenti. In gravida interdum lacinia.
+      </div>
+      <div className="playthrough__sentence">
+        Click the &quot;Try Again&quot; button. To mess arround with your Fog
```

Comment:
Nitpick: "Click the Try Again button to continue editing your Fog Index Calculator"